app/renderer/lib/tabUtil.js

```diff
@@ -36,3 +36,8 @@ module.exports.hasBreakpoint = (breakpoint, arr) => {
   arr = Array.isArray(arr) ? arr : [arr]
   return arr.includes(breakpoint)
 }
+
+module.exports.hasTabAsRelatedTarget = (event) => {
+  const tabComponents = /tab(?=_|area|title|id)|closetab|icon/i
```

Comment:
I ended up checking for `parentNode` instead + nulling pointer events for elements I don't want to be triggered. That method is now tested too.